e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -799,6 +788,39 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
       horizontal: -100,
     });
   });
+
+  it("should not crash notebook when metric is used as an aggregation and breakout is applied (metabase#40553)", () => {
+    createMetric({
+      name: "Revenue",
+      description: "Sum of orders subtotal",
+      table_id: ORDERS_ID,
+      definition: {
+        "source-table": ORDERS_ID,
+        aggregation: [["sum", ["field", ORDERS.SUBTOTAL, null]]],
+      },
+    }).then(({ body }) => {
+      cy.wrap(body.id).as("metricId");
+    });
+
+    cy.get("@metricId").then(metricId => {
+      const questionDetails = {
+        query: {
+          "source-table": ORDERS_ID,
+          breakout: [["field", ORDERS.CREATED_AT, null]],
+          aggregation: [["metric", metricId]],
+        },
+      };
+      cy.createQuestion(questionDetails, { visitQuestion: true });
+    });
+
+    openNotebook();
+
+    getNotebookStep("summarize").contains("Revenue").click();
+
+    popover().within(() => {
+      cy.findByTestId("expression-editor-textfield").contains("[Revenue]");
+    });
```

Comment:
Nit:
```suggestion
    popover().findByTestId("expression-editor-textfield").contains("[Revenue]");
```

Within should be used only when there are multiple assertions in the block. It's perfectly fine to just chain `findByTestId` directly here.